jobs/ml_jobs/artist_linkage/pyproject.toml

```diff
@@ -28,6 +29,13 @@ dependencies = [
     "unidecode>=1.3.8",
 ]
 
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src", "scripts"]
+
 ## ruff
 # Recommended ruff config for now, to be updated as we go along.
```

Comment:
Question: Can we improve lint and code quality overall as it is a newer job ? Also can we set py12 per default ?